sdformat_to_mjcf/tests/resources/double_pendulum.sdf

```diff
@@ -0,0 +1,208 @@
+<?xml version='1.0'?>
+<sdf version="1.4">
+  <model name="double_pendulum_with_base">
```

Comment:
This hasn't been merged yet. I think we should limit this PR to files that have been reviewed and merged.